content/docs/guides/node-exporter.md

```diff
@@ -0,0 +1,148 @@
+
+```
+INFO[0000] Starting node_exporter (version=0.16.0, branch=HEAD, revision=d42bd70f4363dced6b77d8fc311ea57b63387e4f)  source="node_exporter.go:82"
+INFO[0000] Build context (go=go1.9.6, user=root@a67a9bc13a69, date=20180515-15:53:28)  source="node_exporter.go:83"
+INFO[0000] Enabled collectors:                           source="node_exporter.go:90"
+INFO[0000]  - boottime                                   source="node_exporter.go:97"
+...
+INFO[0000] Listening on :9100                            source="node_exporter.go:111"
+```
+
+## Node exporter metrics
+
+Once the node exporter is installed and running, you can verify that metrics are being exported by cURLing the `/metrics` endpoint:
+
+```bash
+curl http://localhost:9100/metrics
+```
+
+You should see output like this:
+
+```
+# HELP go_gc_duration_seconds A summary of the GC invocation durations.
+# TYPE go_gc_duration_seconds summary
+go_gc_duration_seconds{quantile="0"} 3.8996e-05
+go_gc_duration_seconds{quantile="0.25"} 4.5926e-05
+go_gc_duration_seconds{quantile="0.5"} 5.846e-05
+# etc.
+```
+
+Success! The node exporter is now exposing a wide variety of system metrics that Prometheus can scrape.
+
+## Configuring your Prometheus instances
+
+Your locally running Prometheus instance needs to be properly configured in order to access node exporter metrics. The following [`scrape_config`](../prometheus/latest/configuration/configuration/#<scrape_config>) block will tell Prometheus that scrape from the node exporter via `localhost:9100`:
+
+```yaml
+scrape_configs:
+- job_name: 'node'
+  static_configs:
+  - targets: ['localhost:9100']
+```
+
+Once Prometheus is [installed](../../introduction/first_steps) you can start it up, using the `--config.file` flag to point to the Prometheus configuration that you created:
+
+```bash
+prometheus --config.file=./prometheus.yml
+```
+
+## Exploring node exporter metrics through the Prometheus expression browser
+
+Now that Prometheus is scraping metrics from a running node exporter instance, we can explore those metrics using the Prometheus UI (aka the [expression browser](/docs/visualization/expression-browser)).
+ Navigate to `localhost:9090/graph` in your browser. Metrics specific to the node exporter are prefixed with `node_` and include metrics like `node_cpu_seconds_total` and `node_exporter_build_info`.
+
+To see all metrics available for the `node` job:
+
+```
+curl http://localhost:9100/metrics
+```
+
+The `node_filesystem_avail_bytes` metric, for example, informs you how much disk space is available to non-root users on each filesystem.
+
+```
+node_filesystem_avail_bytes{job="node"}
+```
+
+This will likely bring up metrics for a variety of different `device`s and `mountpoint`s. Here's an example output:
+
+```
+node_filesystem_avail_bytes{device="/dev/sda1",fstype="ext4",instance="node:9100",job="node",mountpoint="/etc/hostname"}	  15077224448
```

Comment:
This is still atypical output, as it is from inside docker.

The instance label also doesn't match the configuration